lib/main_publiccloud.pm

```diff
@@ -163,10 +163,11 @@ sub load_publiccloud_tests {
     }
     else {
         loadtest 'boot/boot_to_desktop';
-        if (check_var('PUBLIC_CLOUD_DOWNLOAD_TESTREPO', 1)) {
+        if (get_var('PUBLIC_CLOUD_MIGRATION')) {
+            loadtest('publiccloud/migration');
+        } elsif (check_var('PUBLIC_CLOUD_DOWNLOAD_TESTREPO', 1)) {
             load_publiccloud_download_repos();
```

Comment:
Where will the migration scenario be running? If in maintenance shouldn't we run this after the system gets the maintenance updates?

Reply:
it should not get maintenance updates because we testing unreleased migration package so better to keep other pieces as stable as possible

Reply:
Ah, yes, I was little confused. Sorry.